build.sh

```diff
@@ -71,6 +72,23 @@ function hasArg {
     (( ${NUMARGS} != 0 )) && (echo " ${ARGS} " | grep -q " $1 ")
 }
 
+function cmakeArgs {
+    # Check for correctly formatted cmake args option
+    if [[ $(echo $ARGS | grep -E "\-\-cmake\-args=\"") ]]; then
+        CMAKE_ARGS="$(echo $ARGS | grep -Eo "\-\-cmake\-args=\".+\" ")"
+        if [[ -n ${CMAKE_ARGS} ]]; then
+	    # Remove the full  CMAKE_ARGS argument from list of args so that it passes validArgs function
+	    ARGS=${ARGS//$CMAKE_ARGS/}
+	    # Filter the full argument down to just the extra string that will be added to cmake call
+            CMAKE_ARGS=$(echo $CMAKE_ARGS | grep -Eo "\".+\"" | sed -e 's/^"//' -e 's/"$//')
```

Comment:
The cmakeArgs function uses a mixture of tabs and spaces for formatting causing bad formatting on at least lines 83 and 88